Classes/Controllers/PBGitHistoryController.m

```diff
@@ -590,19 +590,20 @@ - (void)showCommitsFromTree:(id)sender
 - (void) checkoutFiles:(id)sender
 {
 	NSMutableArray *files = [NSMutableArray array];
-	for (NSString *filePath in [sender representedObject])
+	for (NSString *filePath in [sender representedObject]) {
```

Comment:
Sorry, it was late, and the churn was getting hard to process.

I know the current style is messy (is there any code which isn't), I don't have an opinion on what to do about that yet. It's just distracting to read unrelated whitespace-bracing changes while you look at +667/-447 lines.

👍 for clang-format. If you have the will to make it work 😉. I'm mostly in-line with the GitHub Objective-C Style Guide.